### DISCUSSION TITLE
'ltsp image' can not import vmlinuz

### DISCUSSION THREAD

**Discussion by james-chi** [ltsp/ltsp#274] [Q&A]
Hi,
After executing command 'ltsp image /' as a root at Ubuntu 20.04 LTS Desktop with most recent update, it reports:
Could not locate vmlinuz and initrd.img in /srv/ltsp/images/x86_64.img

And this is my /boot directory list (partial):
lrwxrwxrwx 1 root root       27 May  5 15:45 initrd.img -> initrd.img-5.4.0-29-generic
-rw-r--r-- 1 root root 84268824 May  5 15:16 initrd.img-5.4.0-26-generic
-rw-r--r-- 1 root root 84344420 May  5 15:53 initrd.img-5.4.0-29-generic
lrwxrwxrwx 1 root root       27 May  5 15:06 initrd.img.old -> initrd.img-5.4.0-26-generic
...
lrwxrwxrwx 1 root root       24 May  5 15:45 vmlinuz -> vmlinuz-5.4.0-29-generic
-rw-r--r-- 1 root root 11657976 May  5  2020 vmlinuz-5.4.0-26-generic
-rw------- 1 root root 11657976 Apr 29 22:54 vmlinuz-5.4.0-29-generic
lrwxrwxrwx 1 root root       24 May  5 15:45 vmlinuz.old -> vmlinuz-5.4.0-26-generic

Removing symbolic link doesn's solve the problem.
Any help or new code?

**Comment by alkisg** [comment]
Hi James,

see my [comment there](https://github.com/ltsp/ltsp/issues/43#issuecomment-541269010).

**Comment by james-chi** [comment]
alkisg, thanks. it solved my issue.

**Comment by alkisg** [comment]
Great; btw, I moved the issue to the community discussion tracker, as it wasn't an LTSP bug.
